impl/client/src/main/javascript/web/pentaho/visual/ccc/barAbstract/model.js

```diff
@@ -18,24 +18,25 @@ define([
   "../categoricalContinuousAbstract/model",
   "pentaho/i18n!../abstract/i18n/model",
   "../abstract/mixins/settingsMultiChartType"
-], function(module, categoricalContinuousAbstractModelFactory, bundle, settingsMultiChartType) {
+], function(module, baseModelFactory, bundle, settingsMultiChartType) {
```

Comment:
Maybe keep `ccc` as a top umbrella folder... So they can possible be distributed in a separate bundle.

I think this can be a great move! It would better explain the _why_ of the model / view separation.

I even think our doc examples should start with an example only implementing a view and reusing an existing model, and a more advanced example where the developer needs to create its own model.

(in fact, as we don't have a viz registry but a model registry, both need to have their own model so the viz appears? but the first can simply extend one?)